test/client/query-subscribe.js

```diff
@@ -2,7 +2,7 @@ var expect = require('expect.js');
 var async = require('async');
 var util = require('../util');
 
-module.exports = function() {
+module.exports = function(makeSortedQuery) {
```

Comment:
Make this an options object for future proofing. Good chance we'll need other generic query building methods in the future and will be better if passed in as a config object than a bunch of arguments.

```